docfx_yaml/markdown_utils.py

```diff
@@ -301,6 +301,16 @@ def move_markdown_pages(
     # Used to keep track of the index page entry to insert later.
     index_page_entry = None
 
+    list_of_markdown_file_names = [
```

Comment:
Use a set instead of a list since this is used to test existence and order doesn't matter?

Reply:
Done.